src/cpdetect/compiler_detector/heuristics/pe_heuristics.cpp

```diff
@@ -551,6 +635,63 @@ void PeHeuristics::getSecuROMHeuristics()
 	}
 }
 
+/**
+ * Detection of MPRMMGVA packer
+ */
+void PeHeuristics::getMPRMMGVAHeuristics()
+{
+	const uint8_t * fileData = reinterpret_cast<const uint8_t *>(search.getPlainString().c_str());
+	const uint8_t * filePtr = fileData + toolInfo.epOffset;
+	const uint8_t * fileEnd = fileData + search.getPlainString().length();
```

Comment:
I suggest storing the result of `search.getPlainString()` into a variable:
```cpp
const auto & content = search.getPlainString();
const uint8_t * fileData = reinterpret_cast<const uint8_t *>(content.c_str());
const uint8_t * filePtr = fileData + toolInfo.epOffset;
const uint8_t * fileEnd = fileData + content.length();
```
Reasons:
* To make it apparent that `search.getPlainString()` returns a reference. Otherwise, if it returned the string by value, dereferencing `fileData` would result into undefined behavior (most probably a crash).
* No need to call `search.getPlainString()` twice.